Troubleshooting/resolving-peer-rejected.md

```diff
@@ -11,9 +17,12 @@ On the rejected peer:
     UUID file)
 3.  Start glusterd
 4.  Probe one of the good peers
+
+                gluster peer probe <server>
+
 5.  Restart glusterd, check 'gluster peer status'
 6.  You may need to restart glusterd another time or two, keep checking
```

Comment:
Just a side note: From a user's perspective, statements in line 24 to 28 does not instil confidence in Gluster :(

Reply:
That was the existing documentation - I just moved it to a different location. I plan to work with the glusterd folks to get it done properly.